projects/aca-testing-shared/src/utilities/user-actions.ts

```diff
@@ -222,4 +235,45 @@ export class UserActions {
       }
     } else Logger.error('\t>>> ', response);
   }
+
+  async waitForRecentFiles(data: { expect: number }) {
+    try {
+      const nodes = async () => {
+        const totalItems = (await this.getRecentItems()).length;
+        if (totalItems !== data.expect) {
+          return Promise.reject(totalItems);
+        } else {
+          return Promise.resolve(totalItems);
+        }
+      };
+
+      return await Utils.retryCall(nodes);
+    } catch (error) {
+      Logger.error(`SearchApi waitForRecentFiles : catch : `);
+      Logger.error(`\tExpected: ${data.expect} items, but found ${error}`);
+    }
+  }
+
+  async getRecentItems(): Promise<ResultSetRowEntry[]> {
```

Comment:
this already exists here https://github.com/Alfresco/alfresco-content-app/blob/develop/projects/aca-testing-shared/src/utilities/repo-client/apis/search/search-api.ts#L38

and the wait method for this is this https://github.com/Alfresco/alfresco-content-app/blob/develop/projects/aca-testing-shared/src/utilities/repo-client/apis/search/search-api.ts#L114

Reply:
IIRC, the above link is deprecated, right?